lib/routes/account.js

```diff
@@ -410,6 +410,8 @@ module.exports = function (
           })
         }
 
+        request.emitMetricsEvent('flow.signin.submit')
```

Comment:
Unfortunately this clashes with an event name we're already emitting from the content server. See the full list [here](https://github.com/mozilla/fxa-auth-server/blob/phil/issue-1366/docs/metrics-events.md#flow-events).

I think it's best if we reserve all `flow.*` names for the content server, seeing as that is how things have panned out so far and the distinction may help us avoid clashes in the future. Having a think about what I think this event should be now...